source/_docs/composer-drupal-8.md

```diff
@@ -9,6 +9,16 @@ contributors:
 
 [Composer](https://getcomposer.org/) is a dependency management tool that allows PHP code to be more easily shared across projects. As Drupal 8 was written, many of its subsystems written specifically for Drupal were replaced by more widely used packages. For instance, Drupal's notoriously unwieldy function for making external requests, `drupal_http_request()`, [was replaced](https://www.drupal.org/node/1862446) by [Guzzle](http://guzzlephp.org/). Guzzle is included in Drupal 8 via Composer.
 
+Before using Composer to manage your site development you should choose a workflow. By default, Pantheon controls and distributes core updates via git commits, which conflicts with updates processed by Composer. We suggest developers using Composer install their sites with the [Example Drops 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer), which moves Drupal to a "nested docroot". The repository README file covers the two common methods of deployment.
+
+<div class="alert alert-info">
+<h4 class="info">Note</h4><p markdown="1">
+The methods described by the Example Drops 8 Composer repository replace the standard Drupal 8 upstream. By choosing this method, you become responsible for ensuring that your core remains up to date, per our [scope of support](/docs/getting-support/#scope-of-support)
```

Comment:
Previously, the user is responsible for clicking the dashboard button to accept updates provided by Pantheon. With this method, the user is responsible for running `composer update` to accept updates resolved by Composer. Analogous to the comment in the "scope-of-support", if the user selects a non-Pantheon site template (e.g. drupal-composer/drupal-project), then their site will not work correctly. However, if the user selects a Pantheon site template, then Composer will provide the core files. The dashboard will still offer to provide updates. If the user applies them, then bad things will happen.